Gemfile

```diff
@@ -15,10 +15,12 @@ gem 'sass-rails',                 '6.0.0'
 gem 'webpacker',                  '4.2.2'
 gem 'turbolinks',                 '5.2.1'
 gem 'jbuilder',                   '2.10.0'
+# pg был только в продакшне
```

Comment:
Do not use Russian in sources, it limits people who can read them with Russian-speaking only